frontend/www/js/omegaup/components/course/ViewProgress.vue

```diff
@@ -1,39 +1,62 @@
 <template>
-  <div class="omegaup-course-viewprogress card">
-    <div class="card-header">
-      <h2>
-        <a v-bind:href="courseUrl">{{ course.name }}</a>
-      </h2>
-    </div>
-    <div class="card-body">
-      <table class="table table-striped">
-        <thead>
-          <tr>
-            <th>{{ T.wordsName }}</th>
-            <th class="score" v-for="assignment in assignments">
-              {{ assignment.name }}
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr v-for="student in students">
-            <td>
-              <a v-bind:href="studentProgressUrl(student)">{{
-                student.name || student.username
-              }}</a>
-            </td>
-            <td class="score" v-for="assignment in assignments">
-              {{ Math.round(score(student, assignment)) }}
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-    <!-- panel-body -->
-    <div class="card-footer">
-      {{ T.courseStudentsProgressExportToSpreadsheet }}:
-      <a v-bind:download="csvFilename" v-bind:href="csvDataUrl">.csv</a>
-      <a v-bind:download="odsFilename" v-bind:href="odsDataUrl">.ods</a>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-10">
+        <div class="omegaup-course-viewprogress card">
+          <div class="card-header">
+            <h2>
+              <a v-bind:href="courseUrl">{{ course.name }}</a>
+            </h2>
+          </div>
+          <div class="card-body table-responsive">
+            <table class="table table-striped">
+              <thead>
+                <tr>
+                  <th>{{ T.wordsName }}</th>
+                  <th class="score" v-for="assignment in assignments">
+                    {{ assignment.name }}
+                  </th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr v-for="student in students">
+                  <td>
+                    <a v-bind:href="studentProgressUrl(student)">{{
+                      student.name || student.username
+                    }}</a>
+                  </td>
+                  <td class="score" v-for="assignment in assignments">
+                    {{ Math.round(score(student, assignment)) }}
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+      </div>
+      <div class="col-sm-2">
+        <div class="card sticky-top sticky-offset">
```

Comment:
```suggestion
        <div class="card sticky-top mt-3">
```

algo así funciona para evitar declarar esa clase extra?

Reply:
No funciona, por que el texto deja de verse al hacer el scrooll debido a la navbar, y esa clase declarada, hace que sea su posicionamiento sea respecto al viewport.